icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -201,14 +201,6 @@ ant copy-cldr-testdata
 # and add the following line to the beginning of these two files
 # # File copied from cldr common/testData/localeIdentifiers/localeCanonicalization.txt
 
-# 4e. Temporary fix until ICU-21496 is addressed: Need to remove bogus content.
-cd $ICU4C_DIR/source/data
-open rbnf/nb.txt
```

Comment:
Just remove the file. Don't use the "specials" to try and define parent relations.

And why not move the file?

Reply:
Huh? Not sure what you are talking about here. We do need to have data/rbnf/nb.txt, but now it is generated correctly.

If you are talking about the data/xml/rbnf files for Norwegian, they are *not* being used to define parent relationships, that is being done in the CLDR parent data. They *are* being used to define ICU-specific RBNF additions; but now that additional data is only needed for "no", and inherited by "nb" and "nn".